core/src/nativeTest/kotlin/com/saveourtool/processbuilder/ProcessBuilderInternalTest.kt

```diff
@@ -0,0 +1,47 @@
+package com.saveourtool.processbuilder
+
+import com.saveourtool.processbuilder.utils.CurrentOs
+import com.saveourtool.processbuilder.utils.getCurrentOs
+import okio.FileSystem
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+@Suppress("LOCAL_VARIABLE_EARLY_DECLARATION",
+    "SAY_NO_TO_VAR",
```

Comment:
where is `var`?

Reply:
It used to be here, but later test were a bit reworked

Reply:
Removed it